### ISSUE TITLE
Bacula_jobs miss State "Warning"

### ISSUE THREAD

**Issue by Jounixjo** [Bastian-Kuhn/Check_MK-Treasures#30]
State Warning is not covered by bacula.py plugin and bacula_jobs. Therefore this State results always in a Warning and can not be configured within CMK GUI.
Adding following line will solve it. Tested in Check MK 2.0.0p9 and 1.6.0p17.

    ('W',  "Terminated with warnings"),

bacula_job_states = [
    ('A',  "Canceled by user"),
    ('B',  "Blocked"),
    ('C',  "Created, but not running"),
    ('c',  "Waiting for client resource"),
    ('D',  "Verify differences"),
    ('d',  "Waiting for maximum jobs"),
    ('E',  "Terminated in error"),
    ('e',  "Non-fatal error"),
    ('f',  "fatal error"),
    ('F',  "Waiting on File Daemon"),
    ('j',  "Waiting for job resource"),
    ('M',  "Waiting for mount"),
    ('m',  "Waiting for new media"),
    ('p',  "Waiting for higher priority jobs to finish"),
    ('R',  "Running"),
    ('S',  "Scan"),
    ('s',  "Waiting for storage resource"),
    ('T',  "Terminated normally"),
    ('t',  "Waiting for start time"),
]

**Comment by Bastian-Kuhn** [comment]
Hi, thank you very much for this Update.

Would it be OK for you, to update the last digit on the version (info file in src), and use the packer command to create also an updated MKP file? Then the Pull Request would be complete, and I would not need to intervene? 

regards,
Bastian

**Comment by Jounixjo** [comment]
Hi,

I were able to create the MKP file and upload it in another Pull request. I am sorry, I am not that familiar with GIT and pull requests in such detail yet, therefore there exist three pull requests now.

Thanks four your work with Check MK.

Mit freundlichen Grüßen / Kind regards

Johannes Wendel
________________________________

**Comment by Bastian-Kuhn** [comment]
Thank you very much. 
Don't worry, I merged them all.
